stories/rating-indicator/rating-indicator.stories.js

```diff
@@ -0,0 +1,433 @@
+    <div class="example-container">
+        <span>Default:</span>
+        <div class="fd-rating-indicator fd-rating-indicator--hide-dynamic-text">
+            <div class="fd-rating-indicator__container" aria-label="Rating of this item is 2 out of 5">
+                <input aria-label="1 star" type="radio" class="fd-rating-indicator__input" id="rating-1" name="rating" value="1">  
+                <label class="fd-rating-indicator__label" for="rating-1"></label>
+                
+                <input aria-label="2 star" type="radio" class="fd-rating-indicator__input" id="rating-2" name="rating" value="2" checked>
+                <label class="fd-rating-indicator__label" for="rating-2"></label>
+    
+                <input aria-label="3 star" type="radio" class="fd-rating-indicator__input" id="rating-3" name="rating" value="3">
+                <label class="fd-rating-indicator__label" for="rating-3"></label>            
+    
+                <input aria-label="4 star" type="radio" class="fd-rating-indicator__input" id="rating-4" name="rating" value=4">
+                <label class="fd-rating-indicator__label" for="rating-4"></label>
+    
+                <input aria-label="5 star" type="radio" class="fd-rating-indicator__input" id="rating-5" name="rating" value="5">
+                <label class="fd-rating-indicator__label" for="rating-5"></label>
+            </div>
+            <span class="dynamic-text">(2 of 5)</span>
+        </div>
+    </div>
+    
+    <div class="example-container">
+        <span>XS:</span>
+        <div class="fd-rating-indicator fd-rating-indicator--xs">
+            <div class="fd-rating-indicator__container" aria-label="Rating of this item is 2 out of 5">
+                <input aria-label="1 star" type="radio" class="fd-rating-indicator__input" id="rating-xs-1" name="rating-xs" value="1">  
+                <label class="fd-rating-indicator__label" for="rating-xs-1"></label>
+    
+                <input aria-label="2 star" type="radio" class="fd-rating-indicator__input" id="rating-xs-2" name="rating-xs" value="2" checked>
+                <label class="fd-rating-indicator__label" for="rating-xs-2"></label>
+                
+                <input aria-label="3 star" type="radio" class="fd-rating-indicator__input" id="rating-xs-3" name="rating-xs" value="3">
+                <label class="fd-rating-indicator__label" for="rating-xs-3"></label>
+                
+                <input aria-label="4 star" type="radio" class="fd-rating-indicator__input" id="rating-xs-4" name="rating-xs" value=4">
+                <label class="fd-rating-indicator__label" for="rating-xs-4"></label>
+    
+                <input aria-label="5 star" type="radio" class="fd-rating-indicator__input" id="rating-xs-5" name="rating-xs" value="5">
+                <label class="fd-rating-indicator__label" for="rating-xs-5"></label>
+            </div>
+
+            <span class="dynamic-text">(2 of 5)</span>
```

Comment:
You can't have naming like this. If a project using fundamentals have a class dynamic-text there gonna be a styling conflict. We prefix our classes with `fd-` to avoid this. If this element is part of the Rating indicator it should be `fd-rating-indicator__dynamic-text`. If it's a stand-alone element it's styling should be in another file

Reply:
fixed